service/resource/configmap/update_test.go

```diff
@@ -558,13 +560,16 @@ func Test_Resource_ConfigMap_Reload(t *testing.T) {
 		t.Fatalf("error returned updating configmap: %s\n", err)
 	}
 
+	// Wait out the rate limit
+	time.Sleep(2 * time.Second)
```

Comment:
I do not like time based tests that much. I do not have a better solution for the problem right now though. Can we anyway try to decrease this a bit? I find it very odd to wait a few seconds when running unit tests. IMO having 100 and 200 milliseconds should work as well while being 10 times faster.